src/Classes/Cli/HelpRenderer.php

```diff
@@ -126,24 +117,47 @@ private function renderOptions(): string
             return '';
         }
 
-        $arguments = \PHP_EOL;
-        $arguments .= TextRenderer::color('Options:', TextRenderer::COLOR_YELLOW) . \PHP_EOL;
+        $options = \PHP_EOL;
```

Comment:
Let the caller decide whether he wants an empty line or not. Do this in render()